src/pages/workspaces/workspace/jobHistory/WorkflowDashboard.js

```diff
@@ -0,0 +1,236 @@
+  }
+
+  return div({ style: { padding: '1rem 2rem 2rem', flex: 1, display: 'flex', flexDirection: 'column' } }, [
+    div({ style: { marginBottom: '1rem', display: 'flex', alignItems: 'center' } }, [
+      h(Link, {
+        href: Nav.getLink('workspace-job-history', { namespace, name }),
+        style: { alignSelf: 'flex-start', display: 'inline-flex', alignItems: 'center', padding: '0.5rem 0' }
+      }, [icon('arrowLeft', { style: { marginRight: '0.5rem' } }), 'Job History']),
+      div({ style: { margin: '0 0.5rem' } }, ['>']),
+      h(Link, {
+        href: Nav.getLink('workspace-submission-details', { namespace, name, submissionId }),
+        style: { alignSelf: 'flex-start', display: 'flex', alignItems: 'center', padding: '0.5rem 0' }
+      }, [`Submission ${submissionId}`]),
+      div({ style: { margin: '0 0.5rem' } }, ['>']),
+      div({ style: Style.elements.sectionHeader }, [`Workflow ${workflowId}`])
+    ]),
+    _.isEmpty(workflow) ? centeredSpinner() : div({ style: { display: 'flex', flexWrap: 'wrap' } }, [
+      makeSection('Workflow Status', [
+        div({ style: { lineHeight: '24px' } }, [makeStatusLine(style => statusIcon(status, style), status)])
+      ]),
+      makeSection('Workflow Timing', [
+        table({ style: { marginTop: '0.3rem', lineHeight: '20px' } }, [
+          tbody([
+            tr([td({ style: styles.sectionTableLabel }, ['Start:']), start ? td([Utils.makeCompleteDate(start)]) : 'N/A']),
+            tr([td({ style: styles.sectionTableLabel }, ['End:']), end ? td([Utils.makeCompleteDate(end)]) : 'N/A'])
+          ])
+        ])
+      ]),
+      makeSection('Links', [
+        div({ style: { display: 'flex' } }, [
+          h(Link, {
+            ...Utils.newTabLinkProps,
+            href: `${getConfig().jobManagerUrlRoot}/${workflowId}`,
+            style: { padding: '0.6rem' },
+            tooltip: 'Job Manager'
+          }, [
+            div({ style: { display: 'flex', alignItems: 'center' } }, [
+              icon('tasks', { size: 18 }),
+              span([' Job Manager'])
+            ])
+          ]),
+          h(Link, {
+            ...Utils.newTabLinkProps,
+            href: bucketBrowserUrl(`${bucketName}/${submissionId}/${workflowName}/${workflowId}`),
+            style: { padding: '0.6rem' },
+            tooltip: 'Execution directory'
+          }, [
+            div({ style: { display: 'flex', alignItems: 'center' } }, [
+              icon('folder-open', { size: 18 }),
+              span([' Execution Directory'])
+            ])
+          ]),
+          h(Link, { onClick: () => setShowLog(true), style: { padding: '.6rem' } }, [
+            icon('fileAlt', { size: 18, style: { marginRight: '0.5rem' } }),
+            ' View execution log'
+          ])
+        ])
+      ]),
+      makeSection('Call Statuses', [
+        workflow.calls && false ? statusCell(workflow) : div({ style: { padding: '0.6rem' } }, ['No calls have been started by this workflow.'])
```

Comment:
We should either enable or remove this.

Reply:
Oops, this was me testing that it looked alright if the workflow had no calls. Wasn't intended to be committed 🤦